app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -89,7 +89,8 @@ import org.simple.clinic.util.room.UuidRoomTypeConverter
     PatientStatus.RoomTypeConverter::class,
     ReminderConsent.RoomTypeConverter::class,
     BloodSugarMeasurementType.RoomTypeConverter::class,
-    DeletedReason.RoomTypeConverter::class
+    DeletedReason.RoomTypeConverter::class,
+    User.CapabilityType.TypeAdapter::class
```

Comment:
~~We need to add `RoomTypeConverter` here.~~ NVM it has been added in next commit